sdk/python/v1beta1/setup.py

```diff
@@ -20,8 +22,18 @@
     "setuptools>=21.0.0",
     "urllib3>=1.15.1",
     "kubernetes>=23.6.0",
+    "grpcio==1.41.1",
 ]
 
+katib_grpc_api_file = "kubeflow/katib/katib_api_pb2.py"
+
+# Copy Katib gRPC Python APIs to use it in the Katib SDK Client.
+# We need to copy this file only on the SDK building stage, not on installation stage.
+if not os.path.exists(katib_grpc_api_file):
```

Comment:
How do we ensure files are uptodate and in sync?

Reply:
@johnugeorge It's a good point.

Maybe instead of checking that file `kubeflow/katib/katib_api_pb2.py` exists, we should check if `../../../pkg/apis/manager/v1beta1/python/api_pb2.py` file exists and always replace it with the newest ?

```python
katib_grpc_api_file = "../../../pkg/apis/manager/v1beta1/python/api_pb2.py"

# Copy Katib gRPC Python APIs to use it in the Katib SDK Client.
# We need to copy this file only on the SDK building stage, not on installation stage.
if os.path.exists(katib_grpc_api_file):
    shutil.copy(
        katib_grpc_api_file, "kubeflow/katib/katib_api_pb2.py",
    )
```

In that case, on the SDK building stage we always copy the newest gRPC APIs file ?

Any other suggestions @johnugeorge @tenzen-y ?

Reply:
Yeah. I think, "always copy" is the safer solution

Reply:
> In that case, on the SDK building stage we always copy the newest gRPC APIs file?

sgtm